applications/HDF5Application/python_scripts/utils.py

```diff
@@ -0,0 +1,58 @@
+'''HDF5 process utilities.
+
+license: HDF5Application/license.txt
+'''
+import KratosMultiphysics
+
+
+def IsDistributed():
+    return KratosMultiphysics.ParallelEnvironment.GetDefaultDataCommunicator().IsDistributed()
```

Comment:
```suggestion
    return KratosMultiphysics.DataCommunicator.GetDefault().IsDistributed()
```

this is the newer/smplified syntax

Reply:
I didn't want to merge master to do this but now I did it.